File: src/main/java/us/achromaticmetaphor/imcktg/AboutText.java
package us.achromaticmetaphor.imcktg;

import android.app.Activity;
import android.os.Bundle;
import android.text.method.ScrollingMovementMethod;
import android.widget.TextView;

public class AboutText extends Activity {
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_about);
    final TextView about = findViewById(R.id.about);
    about.setMovementMethod(new ScrollingMovementMethod());
    about.setText(getIntent().getExtras().getInt("about_text"));
  }
}

File: src/main/java/us/achromaticmetaphor/imcktg/AboutText.kt
package us.achromaticmetaphor.imcktg

import android.app.Activity
import android.os.Bundle
import android.text.method.ScrollingMovementMethod
import android.widget.TextView

class AboutText : Activity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        findViewById<TextView>(R.id.about).apply {
            movementMethod = ScrollingMovementMethod()
            setText(intent.extras!!.getInt("about_text"))
        }
    }
}